### DISCUSSION TITLE
Detekt on gradle/xml files

### DISCUSSION THREAD

**Discussion by jsouzapp** [detekt/detekt#4095] [Q&A] [answered]
Hello there, anyone know if there is a way to run detekt analysis on build.gradle or .xml files?

**Comment by picklebento** [comment, marked as answer]
Hey @jfsouza-picpay , I don't think this is supported by Detekt since we only checks for .kt files. 
You may want to check out https://github.com/nebula-plugins/gradle-lint-plugin for .gradle files
and [Android lint](https://developer.android.com/studio/write/lint) for .XML files.

**Reply by jsouzapp** [reply]
Thank you soo much :)
